modules/appeals_api/app/swagger/contestable_issues/v0/swagger.json

```diff
@@ -348,8 +352,12 @@
             "authorizationUrl": "https://api.va.gov/oauth2/authorization",
             "tokenUrl": "https://api.va.gov/oauth2/token",
             "scopes": {
-              "appeals/ContestableIssues.read": "Retrieve contestable issues data",
-              "appeals.read": "Retrieve any type of appeal data"
+              "veteran/ContestableIssues.read": "Lets you see info from a your contestable issues",
+              "representative/ContestableIssues.read": "Lets you see info from a veteran's contestable issues",
+              "system/ContestableIssues.read": "Lets you see info from a veteran's contestable issues",
```

Comment:
I wonder if we should be using a more specific word than "you" here, such as calling out the anticipated user in the description (Veteran, Veteran Representative, System).

Reply:
That sounds good - I'll update